content/07.pathogenesis.md

```diff
@@ -1,10 +1,27 @@
-## Pathogenesis and Transmission
+## Genomic and Viral Structure of SARS-CoV-2 in the Context of Pathogenesis, Symptomology, and Transmission
 
 ### Abstract
 
+The novel coronavirus SARS-CoV-2, which emerged in late 2019, has been a major force shaping the year 2020 as it spread around the world infecting tens of millions of people with coronavirus disease 2019 (COVID-19).
+While the viral species was unknown prior to January 2020, its similarity to other coronaviruses that infect humans has allowed for rapid insight into the mechanisms that it uses to infect human hosts, as well as the ways in which the human immune system can respond.
+Here, we contextualize SARS-CoV-2 among other coronaviruses and identify what is known and what can be inferred about its behavior once inside a human host.
+Because the genomic content of coronaviruses is highly conserved and specifies the virus's structure, early genomic analysis provided a significant head start in predicting viral pathogenesis.
```

Comment:
```suggestion
Because the genomic content of coronaviruses, which specifies the virus's structure, is highly conserved, early genomic analysis provided a significant head start in predicting viral pathogenesis.
```